src/sgp/TranslationTable.h

```diff
@@ -10,17 +10,18 @@
 class CTranslationTable
 {
 public:
-	unsigned char m_table[TRANSLATION_TABLE_SIZE];
+	UINT16 m_table[TRANSLATION_TABLE_SIZE];
```

Comment:
With the bigger size and type this will always use over 40x more memory. It's still in the kilo range, so likely it's cleaner to make everyone suffer than to ifdef chinese.

Reply:
May we could malloc the `CTranslationTable::m_table` dynamically.

`UINT16` m_table[0x452] increased 0x452 bytes only，compared to the original.

Reply:
75febe0f50387ab5b29391e460cbe71ecc9e39c9
I change the type of m_table to std::vector<UINT16>.
Hope it can be solved the memory size problem.